composer.json

```diff
@@ -0,0 +1,10 @@
+{
+	"name": "thememix/genesis-translations",
+	"type": "wordpress-plugin",
```

Comment:
Missing the `composer/installers` dev package for this value to be used.

Reply:
Thanks, added to `require` based on instructions at https://github.com/composer/installers#example-composerjson-file.